src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseGradingServiceTest.java

```diff
@@ -110,6 +112,38 @@ public void shouldNotUpdateResultIfNoTestCasesExist() {
         assertThat(result.getScore()).isEqualTo(scoreBeforeUpdate);
     }
 
+    @Test
+    public void shouldAddFeedbackForDuplicateTestCases() {
```

Comment:
this test does not assert the result string and that the instructor notification is created

Reply:
done in afa3191ddf046bbd4a387bcb854c9e356c56f964 f625b9a06fb81358ad8e4877907799f7abdf6d82